README.md

```diff
@@ -64,6 +64,7 @@ Use with carpet mod in the same Minecraft version. Use newer carpet mod versions
 - [stonecutterWood](#stonecutterWood)
 - [treecapitator](#treecapitator)
 - [villagerLeash](#villagerLeash)
+- [playerme](#playerme)
```

Comment:
Da posizionare nel punto corretto, al fine che le regole siano disposte in ordine alfabetico

Reply:
da rinominare in commandPlayerme per consistenza